src/elevenlabs/conversational_ai/conversation.py

```diff
@@ -61,12 +61,18 @@ def __init__(
     ):
         self.extra_body = extra_body or {}
         self.conversation_config_override = conversation_config_override or {}
-        
+
+class ClientTools:
+    """ClientTools provides a way to add custom tools to the conversation."""
```

Comment:
This is pretty vague and the type `Optional[dict]` is not super helpful to figure out what goes here.

Maybe `Optional[dict[str, Callable[[dict], dict]]]` at least? And link to docs / add some example so users know how to populate this.

Reply:
We will add detailed example on documentation on how to use this.